### PR TITLE
chore: disable echarts animation if on a dashboard

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: #7093 

### Description:

Disable `animation` for echarts in dashboards.

Screen recording:

https://github.com/lightdash/lightdash/assets/7611706/4269c893-9161-4ed3-9da4-19edb3f8dd92

Screen recording with pie chart:

https://github.com/lightdash/lightdash/assets/7611706/8feda3c7-c75d-4bdf-a56e-40b40d23a9f2

Screen recording for minimal:

https://github.com/lightdash/lightdash/assets/7611706/acd40b2b-2e56-4f9f-8377-9050fd27907a

Screen recording of individual chart animation still working:

https://github.com/lightdash/lightdash/assets/7611706/acbe1ab9-fe80-4f95-96d1-bee1f9dd8b25

NOTE: animation still works when viewing individual charts

### CODE REVIEW

**Comment by magnew** [comment]
I tried this and I like it. To me it makes things seem faster. 

One thing that would be good to add: piechart config happens separately, we should add the same logic there.

File: packages/frontend/src/hooks/echarts/useEcharts.ts [hunk 1]
```diff
@@ -1435,13 +1439,16 @@ const useEcharts = (validCartesianConfigLegend?: LegendValues) => {
...
             stackedSeries,
-            validCartesianConfig,
-            sortedResults,
+            isInDashboard,
+            validCartesianConfig?.eChartsConfig.legend,
+            validCartesianConfig?.eChartsConfig.grid,
             validCartesianConfigLegend,
+            series,
+            sortedResults,
+            colors,
```
Comment (IrakliJani) [inline packages/frontend/src/hooks/echarts/useEcharts.ts:1451]: there is one more config for pie charts. it's called `useEchartsPieConfig`

**Review by rephus — APPROVED** [review]
Code looks good to me, I assume this also affects /minimal dashboard, right ?

File: packages/frontend/src/hooks/echarts/useEcharts.ts (hunk 1 above)
Comment (joaoviana) [reply packages/frontend/src/hooks/echarts/useEcharts.ts:1451]: niceee, thank you! will do

**Comment by joaoviana** [comment]
> Code looks good to me, I assume this also affects /minimal dashboard, right ?

Sending a video attachment soon 👍

**Pull request merged**